packages/mdc-tabs/README.md

```diff
@@ -284,6 +285,118 @@ import {MDCTabBar, MDCTabBarFoundation} from '@material/tabs';
 const tabBar = new MDCTabBar(document.querySelector('#my-mdc-tab-bar'));
 ```
 
+
+### Using the JavaScript Tab Bar Scroller Component
+
+`mdc-tab-bar-scroller` ships with a Component/Foundation combo for ingesting instances of `mdc-tab-bar`. `mdc-tab-bar-scroller` uses its `initialize()` method call a factory function which gathers and instantiates any tab bar elements that are children of the `mdc-tab-bar-scroller` root element.
+
+The anatomy of `mdc-tab-bar-scroller` includes an instance of `mdc-tab-bar`, forward and back indicators which, when actioned on, move the tab bar left and right, and a scroll frame. The scroll frame is the parent element of the tab bar, and serves to mask the tabs in the tab bar when they overflow the available width.
+
+
+#### Including in code
```

Comment:
We certainly don't _need_ to repeat it, but I think it should stay for two reasons:

1. It doesn't hurt people who already have a tab bar and want to enhance it with scroller now that its available
1. People coming to tab bar from scratch may want to just scroll down to this, then cut and paste for their flavor of tab bar scroller inclusion.